understanding/21/label-in-name.html

```diff
@@ -61,12 +65,17 @@ <h3>Accessible Name and Description Computation specification</h3>
  <p>It is important to note that the specification allows authors to override the name calculated through native semantics. Both <code>aria-label</code> and <code>aria-labelledby</code> take precedence in the name calculation, overriding the visible text as the accessible name even when the visible text label is programmatically associated with the control. For this reason, when a visible label already exists, <code>aria-label</code> should be avoided or used carefully, and <code>aria-labelledby</code> should be used as a supplement with care.</p>
 <p>Finally, <code>aria-describedby</code> is not included in the Accessible Name computation (instead it is part of the Accessible Description computation). By convention, text associated with a control through <code>aria-describedby</code> is announced immediately after the accessible name by screen readers. Therefore, the context of headings, instructions, and group labels can be provided through the accessible description to assist users of screen readers without affecting the experience of those who navigate using speech recognition software.</p>
-			</section>
+</section>
+<h4>Text in parentheses</h4>
+	<section>
+		<p>Note: The term "parenthetical" in this section is describing text that literally appears within a set of round brackets (). It is not used in the more abstract sense of "information related to".</p>
+	<p>It is important to mention parenthetical text in labels in the context of accessible name versus description. In common usage, text in parentheses is considered secondary but relevant to meaning. Users of speech recognition would not typically announce text in parentheses as part of the input name. For that reason, parenthetical text may be optionally considered a description and left out of the accessible name.</p>
+	<p>However, where parenthetical information provides important context, such as indication of a required field or limitations on what is allowed for input, this information must be provided programmatically in some other way to the user if that information is not included as part of the accessible name. For example, "Name (required)" and "Date (YYYY-MM-DD)" could accept "Name" and "Date" as the accessible names. However, in order to pass 1.3.1 Info & Relationships, authors would need to programmatically surface both the required state and the limit on the allowed data formatting (in this example, eight integers fitting the YYYY-MM-DD pattern). The "required" state could be surfaced through the HTML <code>required</code> attribute or by using <code>aria-required="true"</code>. The allowed data formatting could be surfaced by being referenced using the <code>aria-describedby</code>) attribute.</p>
```

Comment:
Is it possible to list outliers to this? In the US, area codes for telephone numbers are sometimes shown within parentheses. Ex: (202)555-5555. The area code should be part of the name.

Reply:
@kengdoj I think this use case falls under the "However, where parenthetical information provides important context, such as indication of a required field or limitations on what is allowed for input." I know how much easier it is top point developers to a line in a spec or documentation, but the possible number of cases that could be an outlier is potentially huge. 
Also, I question any label that includes the formatting. That really should be aria-describedby information.

Reply:
I don't think this text increases understanding. Parenthetical information (relevant, but not necessary to understand), is probably better in the description than the label. Whether it is presented in parentheses (round brackets) or not seems more like a typographical accident.

I'm prone to putting a lot in parentheses. Some use commas and subsidiary clauses instead, others write more sentences.

Reply:
I'd agree with @nattarnoff that the example of the area code in parentheses doesn't seem to enter into the discussion on parentheses in a _label_. @kengdoj can you supply an example of an area code in a phone input label that would be problematic with the guidance listed in this subsection?

Reply:
> Parenthetical information (relevant, but not necessary to understand), is probably better in the description than the label.

@chaals I entirely agree, but there's an unfortunate gap between what people should do and what they DO.  As well as a gap between the intent of this SC and how it is being interpreted These changes were introduced to address issues opened by users, and have support. In fact, this has already been voted on and approved by the WG.